testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml

```diff
@@ -120,9 +120,10 @@ spec:
                 - --metrics-addr=127.0.0.1:8080
                 - --enable-leader-election
                 - --leader-election-id=memcached-operator
+                command:
```

Comment:
Just to follow the standard. I did not add to the migration steps because it is optional.